### ISSUE TITLE
where is the config file?

### ISSUE THREAD

**Issue by nsantini** [kiddkai/atom-node-debugger#179]
silly question, but you mention a config file but not where it is or what to call it

**Comment by madwax** [comment]
Yep I have the same question.  It might be a silly FTM for Atom question.

**Comment by romangusev** [comment]
I cannot find it also

**Comment by codecontemplator** [comment]
You open the file using Atoms menu; file->config...

**Comment by codecontemplator** [comment]
Documentation should be updated to mention this.

**Comment by gauthierm** [comment]
The model VS Code uses where you get a per-project config file is a better approach and would be nice to see here. It may even be possible to steal the file format they are using.

**Comment by codecontemplator** [comment]
Agreed. Project specific settings would be useful. I think the file format should be the same as for the "global" settings used for atom. Only that the project specific overrides the global is set. Seems like atom has no built-in support for this but it should be straight forward to search for a settings file in the project root folder.
